src/delaunay.c

```diff
@@ -92,7 +111,9 @@ static int delaunay_reduce(double red_lattice[3][3], const double lattice[3][3],
 
     lattice_rank = get_extended_basis(basis, lattice, aperiodic_axis);
 
-    for (attempt = 0; attempt < NUM_ATTEMPT; attempt++) {
+    for (attempt = 0; attempt < get_num_attempts(); attempt++) {
```

Comment:
I prefer preparing a variable for `get_num_attempts()` in order not to call this function every loop count.
But, it will be negligible.

Reply:
I think we can make it a global variable later on, but for now this should be ok

Reply:
Oh, now I just re-read what you mean there. I thought you were referring to having a global variable defined when spglib is initialized. You meant instead to have the variable called only once in the for loop. Yeah that's straight-forward, I've fixed that.